.gitignore

```diff
@@ -0,0 +1 @@
+.idea/*
```

Comment:
Please add a new line at the end of this file.

Reply:
Done. I also removed the commented out code from the CSS files